apps/meteor/app/cors/server/cors.ts

```diff
@@ -103,7 +103,7 @@ WebAppInternals.staticFilesMiddleware = function (
 	res.setHeader('Access-Control-Allow-Origin', '*');
 	const { arch, path, url } = WebApp.categorizeRequest(req);
 
-	if (Meteor.isProduction && !cacheControlledRoutes.some((regexp) => regexp.test(path))) {
+	if (Meteor.isProduction && path.match(/\.(js|css|map)$/i) && !cacheControlledRoutes.some((regexp) => regexp.test(path))) {
```

Comment:
Won't this also remove cache from fonts, images and any random file we may be serving?

Reply:
@pierre-lehnen-rc good catch. It will, as it was before. We can fine tune this in another PR